src/main.ts

```diff
@@ -0,0 +1,11 @@
+import { createQuery } from './createQuery'
+
+const { query, useQuery } = createQuery()
+
+export {
+  query,
+  useQuery,
```

Comment:
library exports a default `query` and `useQuery` which is what I'd expect most implementations to use rather than actually calling `createQuery`